### DISCUSSION TITLE
When the new version of the chart for Airflow 2.7.2 will be available?

### DISCUSSION THREAD

**Discussion by quanghungb** [apache/airflow#35369] [Q&A]
Hi 

I can see that there's a commit in master for Airflow 2.7.2 : https://github.com/apache/airflow/blob/47c90cfbded17c4942a80e2941091d37b2d8765d/chart/Chart.yaml#L23

I would like to know when the chart 1.12.0 will be released (approximately)? In fews days? weeks? months? 

Thanks.

**Comment by Taragolis** [comment]
There is no any ETA for releases Airflow and other components, e.g. providers packages, helm chart.
So best way to determine when next version of chart **might** be released it is look at https://airflow.apache.org/docs/helm-chart/stable/release_notes.html

As you could see in general it took from 2 to 4 months

**Reply by potiuk** [reply]
Also nothing holds you back. Our charts can be used for any version of Airflow and if you are waiting for the chart to have 'default' version updated you are basically doing it wrong. Just use latest airflow version in the chart version you have. This is even explain the chart documentation if you look for it

**Reply by quanghungb** [reply]
Yes ... it's written in https://airflow.apache.org/docs/helm-chart/stable/index.html that 
Supported Airflow version: 1.10+, 2.0+

But I was wondering if there are any fixes, additional features are brought by the new version of the chart..

**Reply by potiuk** [reply]
> But I was wondering if there are any fixes, additional features are brought by the new version of the chart..

Well. Your question was specifically for the 2.7.2 version change  not for the new features there. So the answer you got was precisely answering your question about version.

Now, answering your - different and new - question about new features - This is what @Taragolis  answered. We have no fixed timelines. But we release chart with new features every few months.

But let me add as well - you are not blocked here either. You can always look at PRs implemented and use them in your chart if you want. You seem to be curious and wonder, this a way I'd do it if I were you. We absolutely welcome peple who do it and who can even test and contribute fixes to issues way before we release chart and I'd heartily recommend you to help us with that and contribute issues and fixes to the things that aren't yet released. And the more confidence we have from users like you testing it - the more likely is we release it earlier. So a bit the answer to the question of yours is "quite likely it will be released sooner if you help with testing the main version". 

You can always install chart from main branch of ours - we are actually doing that in every pr and our tests are using that "main" version to run Helm and K8S tests so it should be pretty solid.

Also chart is not a black box, it's text based and basically jinja templates - and if you want features earlier - you can cherry-pick them or implement it in your chart independently and make airflow a subchart of yours a and apply modifications in your custom chart to test them. This is the way how you work with charts. You do not expect them to be "black boxes" - you are supposed to modify and adapt them if you need new features. 

Since you are eager to test new thing - are you wiling to follow that path and report back Issues etc? That would ba a great way to contribute back @quanghungb

**Reply by quanghungb** [reply]
Yes my question is specially for the 2.7.2 version because I wanted to try to install the Airflow 2.7.2. As a new user of Airflow I would prefer to use the "official" version of the chart first before trying to do something homemade :) 

BTW thanks all for your replies.

**Reply by potiuk** [reply]
> Yes my question is specially for the 2.7.2 version because I wanted to try to install the Airflow 2.7.2. As a new user of Airflow I would prefer to use the "official" version of the chart first before trying to do something homemade :)

Yes. So use latest chart and change version without wating for the new chart. Again - chart and Airflow are relased independently so if your goal is to install new version of Airflow as soon as it gets released just .... welll... use it with whatever chart you use.
